xml/admin_gui_oa.xml

```diff
@@ -360,7 +360,44 @@ RGW_API_SECRET_KEY="lJzPbZYZTv8FzmJS5eiiZPHxlT2LMGOMW8ZAeOAq"</screen>
     original scale, double-click the chart.
    </para>
   </tip>
+  <para>Within openATTIC there are options to display graphs for longer than
+    15 days. However, by default prometheus only stores history for 15 days.
+    This can be adjusted in <filename>/etc/systemd/system/multi-user.target.wants/prometheus.service</filename>.</para>
+  <procedure>
+    <step>
+      <para>
+        Open <filename>/etc/systemd/system/multi-user.target.wants/prometheus.service</filename>.
+      </para>
+    </step>
+    <step>
+      <para>
+        This file should reference the following:
+      </para>
+      <screen>
+        EnvironmentFile=-/etc/sysconfig/prometheus
+        ExecStart=/usr/bin/prometheus $ARGS
+      </screen>
+      <para>If not does not, add the above two lines and include
+        the following:</para>
+      <screen>
+        ARGS="--storage.tsdb.retention=90d" \
+              --log.level=warn"
+      </screen>
+      <tip>
+        <para>Ensure <literal>ARGS</literal> is a multiline bash string.
+        This enables prometheus to store up to 90 days of data.</para>
+        <para>If you want other time options, the format is as follows:
+          xxh, xxd, xxw, xxy (hours, days, weeks, years).</para>
+      </tip>
+    </step>
+    <step>
+      <para>
+        Restart the prometheus service.
```

Comment:
```suggestion
        Restart the &prometheus: service.
```
Is there a command mentioned to restart the prometheus service afterwards? Maybe it's useful to add it... `systemctl restart ...`